Tests/swift-sdk-swift-tests/RequestCreatorTests.swift

```diff
@@ -29,6 +29,50 @@ class RequestCreatorTests: XCTestCase {
         TestUtils.validateMatch(keyPath: KeyPath(JsonKey.endUnreadMessageCount), value: inboxSession.endUnreadMessageCount, inDictionary: body)
     }
     
+    func testGetInAppMessagesRequestFailure() {
+        let auth = Auth(userId: nil, email: nil)
+        let requestCreator = RequestCreator(apiKey: apiKey, auth: auth)
+        
+        let failingRequest = requestCreator.createGetInAppMessagesRequest(1)
+        
+        if let _ = try? failingRequest.get() {
+            XCTFail("request succeeded despite userId and email being nil")
+        }
+    }
+    
+    func testGetInAppMessagesRequest() {
+        let inAppMessageRequestCount: NSNumber = 42
+        
+        let request = createRequestCreator().createGetInAppMessagesRequest(inAppMessageRequestCount)
```

Comment:
`guard let request = createRequestCreator().createGetInAppMessagesRequest(inAppMessageRequestCount) else {
XCTFail("Could not create getInAppMessagesRequest")
return
}`